README.md

```diff
@@ -2,8 +2,122 @@
 
 The EA Forum is a synced fork of [LessWrong](https://github.com/LessWrong2/Lesswrong2).
 
-## Contributing
+## Technologies
 
-We have not prioritized writing a guide to the tech. You can see the instructions on the LessWrong repo, but any given sentence is more likely than not to be out of date and misleading. If you are at least moderately interested in contributing, message [JP](https://forum.effectivealtruism.org/users/jpaddison). I won't mind the inquiry, and can help you out and in the process might update this readme.
+The EA Forum is built on top of a number major open-source libraries.
 
-We do not use github for issues. Please [contact us](https://forum.effectivealtruism.org/contact) with bug reports.
+1. [Vulcan](http://vulcanjs.org/) is a framework for designing social applications like forums and news aggregators. We started out using it as a library in the usual way, then forked its codebase and diverged considerably.
+
+2. [Typescript](https://www.typescriptlang.org/) is the programming language we're using. It's like Javascript, but with type annotations. We're gradually moving from un-annotated Javascript towards having annotations on everything, and any new code should have type annotations when it's added.
+
+3. [React](https://facebook.github.io/react/) is a user interface programming library developed by Facebook that lets us define interface elements declaratively in the form of components. We use it to define how to render and manage state for all parts of the site.
+
+4. [GraphQL](https://graphql.org/) is a query language that the browser uses to get data from our servers. Most usage of GraphQL is hidden behind utility functions, but occasionally we use it directly to define APIs for accessing and mutating our data.
+
+5. [Apollo](https://www.apollographql.com/docs/) is a client-side ORM which we use for managing data on the client. We interact with it primarily via the React hooks API.
+
+6. [CkEditor5](https://ckeditor.com/) is the default text editor for posts, comments, and some other form fields. [Draft](https://draftjs.org/) is an alternative text editor, which is no longer the default but which we still support.
+
+## Running locally
+
+### Requirements
+
+  * MacOS or Linux
+    * Known to work on MacOS 10.15 and Ubuntu 18.04, should work on others
+  * Node
+    * see `.nvmrc` for the required node version
+    * You can use [Node Version Manager](https://github.com/creationix/nvm) to install the appropriate version of Node
+
+### Installation
+
+Clone our repo:
+
+    git clone git@github.com:centre-for-effective-altruism/EAForum.git
```

Comment:
Would be nice to see this stuff inside an actual `\`\`\`` code fence, so it's clear that it's a shell command.